code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -10,6 +10,10 @@
 		act = copytext(act, 1, t1)
 
 	var/muzzled = is_muzzled()
+	if(muzzled)
+		var/obj/item/clothing/mask/muzzle/M = wear_mask
+		if(M.mute & EMOTE_SOUND)
```

Comment:
Are you sure this is right? `can_speak` below checks for the muzzle blocking sound. Checking for `!(M.mute & EMOTE_VISUAL)` might work better.